### PR TITLE
[New Model]: DaysOfSupply 1.0.0

### LINKED ISSUE #526: [New Model]: DaysOfSupply 1.0.0
## Model Description
We want to create a model to help with projection and analysis of how much of the item can be manufactured from the current supply at hand.
Days of Supply in logistics is a metric used to estimate how long current inventory levels will last under normal consumption patterns. It is determined by dividing the available inventory by the daily usage rate. This calculation is used in supply chain management and it assists in forecasting when new stock is needed, thereby preventing stock shortages or overstocking. It plays a significant role in ensuring efficient inventory turnover, maintaining a balance between having enough stock to meet demand and avoiding excess inventory that ties up capital. 

## Relevant Standards
There are no relevant standards available, however, this standard will be reusing some of existing elements from other Catena-X standards such as relevant inventory item identifiers.

## Example Data

```json
{
  "daysOfSupply": {
    "inventoryItem": {
      "materialGlobalAssetId": "urn:uuid:12345d48-6f1d-47f5-8ded-a441d0d12345",
      "materialNumberCustomer": "MNR-1234-AU123474.001"
    },
    "currentStockLevel": 150,
    "averageDailyUsage": 50,
    "estimatedDaysOfSupplyRemaining": 3
  }
}
``` 

## MS1 Criteria

- [x] The proposed aspect model does not exist already
- [x] The proposed aspect model does not extend an existing aspect but introduces completely new functionality
- [x] The proposal references a Catena-X use case
- [x] Relevant standards are mentioned/linked
- [x] A first draft of the data structure is provided in an example
- [x] A modeling steward has been assigned

### PR DESCRIPTION
**Pull request by stiwan8020**
## Description

Days of Supply in logistics is a metric used to estimate how long current inventory levels will last under normal consumption patterns. Days of Supply metric is used in supply chain management and it assists in forecasting when new stock is needed, thereby preventing stock shortages or overstocking. It plays a significant role in ensuring efficient inventory turnover, maintaining a balance between having enough stock to meet demand and avoiding excess inventory that ties up capital.

 -->

Closes #526 

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.5.1)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

File: io.catenax.days_of_supply/1.0.0/DaysOfSupply.ttl [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+:date a samm:Property ;
+   samm:preferredName "Date of Days of Supply"@en ;
+   samm:description "Date refers to the specific calendar day (current or projected) associated with the measured (or expected) inventory level. It serves as a timestamp for calculating Days of Supply, indicating when the inventory count was taken or projected."@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2024-01-01T14:23:00+01:00"^^xsd:dateTime .
+
+:daysOfSupply a samm:Property ;
+   samm:preferredName "Days of Supply"@en ;
+   samm:description """Amount of days, before the current stock is expected to be exhausted.
```
Comment (agg3fe) [inline io.catenax.days_of_supply/1.0.0/DaysOfSupply.ttl:125]: please correct the spacing

Comment (zanpi8020) [reply io.catenax.days_of_supply/1.0.0/DaysOfSupply.ttl:125]: Hi, thanks for the feedback. Can you elaborate if this is referring to the spacing in the formulas/description of Days of Supply property? Thanks for the review

Comment (stiwan8020) [reply io.catenax.days_of_supply/1.0.0/DaysOfSupply.ttl:108]: We removed the double break line from the description. Did you mean that?

**Review by agg3fe — APPROVED** [review]
LGTM

**Pull request merged**